### ISSUE TITLE
VSCode - long running code has been detected.

### ISSUE THREAD

**Issue by Cyberlane** [wallabyjs/public#995]
### Issue description or question

I am seeing this on my project.
```
[Error] some long running code has been detected: one of your tests is taking more than 5000ms to execute.
```

There is no information about which test is in fact to blame for this though, so it makes debugging it quite a pain, seeing as I have so many tests to begin with. 

### Wallaby.js configuration file

```javascript
module.exports = wallaby => ({
  debug: true,
  files: [
    'src/**/*.js',
    '!src/**/*.test.js',
  ],
  tests: [
    'src/**/*.tests.js',
  ],
  testFramework: 'tape',
  env: {
    type: 'node',
    runner: 'node',
  },
  compilers: {
    '**/*.js': wallaby.compilers.babel(),
  },
  workers: {
    recycle: true,
  },
})
```

### Code editor or IDE name and version

Visual Studio Code v1.9.0

### OS name and version

Windows 7

**Comment by Cyberlane** [comment]
Okay, I have figured out which test was to blame for this, it looked a bit like this.

```javascript
import test from 'tape';

test('some test example', (t) => {
  t('=> specific test info', (assert) => {
    assert.end();
  });
});
```

Take notice to the typo where I call `t()` rather than `t.test()`.  Now my bad test to blame here, however I would have ideally appreciated if the test runner had told me which test was to blame. The way I solved this was by running tape directly in the command line, which then told me that `t` was not a function on this specific test file.
